docs/cn/providers/tencent/components/basic-components/tencent-scf.md

```diff
@@ -79,13 +79,12 @@ myFunction1:
     memorySize: 128
     timeout: 20
     # 打包zip时希望忽略的文件或者目录配置（可选）
-    exclude:
-      - .gitignore
-      - .git/**
-      - node_modules/**
-      - .serverless
-      - .env
-    include:
+    patterns:
+      - '!.gitignore'
+      - '!.git/**'
+      - '!node_modules/**'
+      - '!.serverless'
+      - '!.env'
```

Comment:
If `@servereless/components` uses `serverless`, this will work just as it used to.

This will need to change for the next major version update anyway.
I think that keeping it is creating future trouble.

Do you disagree?

Reply:
> If @servereless/components uses serverless,

It' doesn't use `serverless`, it's com (package internals), it's a completely different CLI